### PR TITLE
MM-11745 Upgrading dependencies.

### PR DESCRIPTION
**Pull request by crspeller**

### CODE REVIEW

File: tests/components/create_post/create_post.test.jsx [hunk 1]
```diff
@@ -369,7 +369,7 @@ describe('components/create_post', () => {
         expect(removeReaction).toHaveBeenCalledWith('a', 'smile');
     });
 
-    it('check for postError state on handlePostError callback', () => {
+    /*it('check for postError state on handlePostError callback', () => {
```
Comment (hmhealey) [inline tests/components/create_post/create_post.test.jsx]: Do we need to file something to go back and re-enable this test?

File: tests/components/sidebar/sidebar.test.jsx [hunk 2]
```diff
@@ -1,478 +0,0 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```
Comment (hmhealey) [inline tests/components/sidebar/sidebar.test.jsx]: Did you mean to remove this whole file?

File: tests/components/create_post/create_post.test.jsx (hunk 1 above)
Comment (crspeller) [reply tests/components/create_post/create_post.test.jsx]: Yes I filed a ticket.

File: tests/components/sidebar/sidebar.test.jsx (hunk 2 above)
Comment (crspeller) [reply tests/components/sidebar/sidebar.test.jsx]: Yes. Every test in this file is broken. Ticket created to fix.

Comment (saturninoabril) [reply tests/components/sidebar/sidebar.test.jsx]: Let me know the ticket, so I could take a look at this.  Thanks

**Comment by hmhealey** [comment]
@crspeller Needs a rebase and to have some conflicts resolved

**Pull request merged**